frontend/src/modules/editor/global/views/editorOriginView.js

```diff
@@ -67,10 +67,21 @@ define(function(require){
       }
     },
 
+    componentPasteLocation: function() {
+      if ($('.block-inner .add-control').length === 0) {
```

Comment:
In adapt the convention is to do an 'early return', and to do if statements on one line if possible, eg the following would be preferred

```
componentPasteLocation: function() {
  if ($('.block-inner .add-control').length > 0) return;
  
  Origin.Notify.alert({
    type: 'error',
    text: 'There are no blocks to paste your copied component into. Please create an empty block and try again.'
  });
  $('.add-control').removeClass('display-none');
},
```